rel/avatar.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: "Link Relation: avatar"
+title: "WebFinger Link Relation: avatar"
```

Comment:
what's the difference between a "WebFinger Link Relation" and just a "Link Relation"?  While these happen to be defined within the webfinger.net namespace, there is nothing necessarily webfinger specific about them.  They're just link relations, right?